src/store/getters.ts

```diff
@@ -1,12 +1,20 @@
-import { RootState } from './RootState';
+import RootState from './RootState';
 import QueryRepresentation from '@/sparql/QueryRepresentation';
 import Property from '@/data-model/Property';
 
 export default {
-	query( { property, value }: RootState ): QueryRepresentation {
-		return { property, value };
+	query( rootState: RootState ): QueryRepresentation {
```

Comment:
Ah yes, this getter is already effectively one half of this two-way transformation between store state and QueryRepresentation. Cool!